Find a functionally equivalent code:android.content.res.Resources.loadColorStateList:COMMENT
Method Modifier: 
Comment:/*package*/

@@ -33,43 +33,43 @@
     if (value.string == null) {
         throw new NotFoundException("Resource is not a ColorStateList (color or path): " + value);
     }
-    String file = value.string.toString();
+    final String file = value.string.toString();
     if (file.endsWith(".xml")) {
         Trace.traceBegin(Trace.TRACE_TAG_RESOURCES, file);
         try {
-            XmlResourceParser rp = loadXmlResourceParser(file, id, value.assetCookie, "colorstatelist");
+            final XmlResourceParser rp = loadXmlResourceParser(file, id, value.assetCookie, "colorstatelist");
             csl = ColorStateList.createFromXml(this, rp);
             rp.close();
         } catch (Exception e) {

